Enterprise-CMCS/cmcs-eregulations
# EREGCSC-2288 -- Subject List ToC for Policy Repository

Resolves [EREGCSC-2288](https://jiraent.cms.gov/browse/EREGCSC-2288)

**Description:**

The policy repository "homepage" displays a list of subjects from which to select.

See [Figma](https://www.figma.com/file/KiWw729Nct6tY9gzmvEkVO/Subject-filter-only?node-id=379%3A104481&mode=dev).

**This pull request changes:**

- Adds Subject "Table of Contents" that is rendered when first visiting the Policy Repository

**Steps to manually verify this change:**

1. All tests pass
2. Visit experimental deployment and see the Subject ToC on Policy Repository page load
3. click a subject in the Subject ToC and make sure page updates with correct subject
